app.rb

```diff
@@ -10,6 +11,18 @@ def initialize
     @rentals = []
   end
 
```

Comment:
- [ ] It would be better if all requests for input and input validation should be placed in the main.rb file. Let us kindly keep in mind best practices and SOLID lesson. Kindly check this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) for more information